### PR TITLE
Release Flux 1.18.0

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,155 @@
...
+### Enhancements
+
+- Disable Image Scanning with `--registry-disable-scanning`
+  fluxcd/flux{[#2745][fluxcd/flux#2745], [#2753][fluxcd/flux#2753]
+  [#2798][fluxcd/flux#2798], [#2813][fluxcd/flux#2813]}
+- Add `scanForFiles` variant of `.flux.yaml` to scan current directory
+  for manifests instead of generating them [fluxcd/flux#2638][]
+- Honor KUBECONFIG env variable in fluxd [fluxcd/flux#2741][]
+  [fluxcd/flux#2760][]
+- Make Kubernetes resource-exclusion configurable through
```
Comment (hiddeco) [inline CHANGELOG.md]: Something went wrong here with the formatting.

Comment (2opremio) [reply CHANGELOG.md]: I don't follow

Comment (hiddeco) [reply CHANGELOG.md]: It should be `fluxcd/flux{[#2749][fluxcd/flux#2749], [#2754][fluxcd/flux#2754]}`

Comment (2opremio) [reply CHANGELOG.md]: Ah, I was confused by the line addressed by the comment.

Comment (2opremio) [reply CHANGELOG.md]: Fixed!

**Comment by stefanprodan** [comment]
@2opremio the "Bump Flux version to 1.18.0" commit should also bump the version in values.yaml.

**Comment by 2opremio** [comment]
Ah, OK. I thought that was done as part of the Chart release.

**Comment by 2opremio** [comment]
done

**Pull request merged**